### PR TITLE
Send one email at the end of committing or revealing

### PR DESCRIPTION
**Pull request by ptare**
Two follow up changes:
1. Environment variables are obviously not the best way to set API keys. Figuring out a better way is tracked in #490 
2. Does the script reveal votes over and over? If so, we might need to update EncryptedSender somehow to remember that the vote was revealed? (Not sure about this yet)

Issue #491

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,6 +14,11 @@ WORKDIR protocol
 RUN npm install
 RUN scripts/buildContracts.sh
 
+# Environment variables to set up email notifications.
+ENV SENDGRID_API_KEY=
```
Comment (mrice32) [inline Dockerfile]: Just curious, what's the purpose of setting them equal to nothing?

File: core/scripts/Voting.js [hunk 2]
```diff
@@ -71,28 +84,43 @@ class VotingSystem {
...
       } else {
-        await this.runReveal(request, roundId);
+        didUpdate = await this.runReveal(request, roundId);
       }
+      if (didUpdate) {
+        updatesMade.push(request);
+      }
+    }
+    if (updatesMade.length > 0) {
+      // TODO(ptare): Get email copies.
```
Comment (mrice32) [inline core/scripts/Voting.js]: What does this mean?

File: Dockerfile [hunk 3]
```diff
@@ -14,6 +14,11 @@ WORKDIR protocol
 RUN npm install
 RUN scripts/buildContracts.sh
 
+# Environment variables to set up email notifications.
```
Comment (mrice32) [inline Dockerfile]: nit: can you add these to the sample `docker run` command at the top?

File: Dockerfile (hunk 1 above)
Comment (ptare) [reply Dockerfile]: Just mostly here as documentation, we could remove these lines if you prefer.

File: core/scripts/Voting.js (hunk 2 above)
Comment (ptare) [reply core/scripts/Voting.js]: copy = actual final text. When we have the email copies, we'll need to plug them in here.

File: Dockerfile (hunk 3 above)
Comment (ptare) [reply Dockerfile]: Done.

File: Dockerfile (hunk 1 above)
Comment (mrice32) [reply Dockerfile]: Nope, just wanted to make sure it was just documentation and not something more subtle :).

File: core/scripts/Voting.js (hunk 2 above)
Comment (mrice32) [reply core/scripts/Voting.js]: Still confused - maybe we can discuss IRL.

Comment (ptare) [reply core/scripts/Voting.js]: Discussed offline, going to fill these in in the next few days.

**Comment by ptare** [comment]
Issue #487

**Pull request merged**